src/main/java/org/mamute/list/TabsHelper.java

```diff
@@ -0,0 +1,47 @@
+package org.mamute.list;
+
+import br.com.caelum.vraptor.http.route.Router;
+import net.vidageek.mirror.dsl.Mirror;
+import org.apache.commons.lang.WordUtils;
+import org.mamute.controllers.ListController;
+
+import javax.inject.Inject;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+public class TabsHelper {
```

Comment:
I personally don't like this approach, seems like you are separating behaviour from attributes, making the Tab model anemic.
I would put those methods at Tab, it should know how to be created and its possible values